### DISCUSSION TITLE
Reshaun's Anime Levels

### DISCUSSION THREAD

**Discussion by reshaun** [Neverball/neverball#330] [Show and tell]
Nevergaiden: The Japanese Levelset.

The Credited to Remake of Super Monkey Ball Games.

Newcommer: Anime marble textures and GUI theme.

<img width="600" height="600" alt="RF-AnimeLevels-V2" src="https://github.com/user-attachments/assets/aeccc8e1-82b9-4744-a0fd-7c73b9ad1953" />

50 Levels in 2 Gaiden sets and 10 Insanity Levels in Gaiden Master pack.

It's Unofficial.

P.S. Please don't sue me, don't judge me, don't suspend me, don't ban me, don't copyright, and don't take down my levelsets forever anymore.

-Reshaun Francis

Special thanks to: SEGA, Super Monkey Ball, and Cutsman

[Reshaun-AnimeLevels-2026-1.6.0.zip](https://github.com/user-attachments/files/30868198/Reshaun-AnimeLevels-2026-1.6.0.zip)

[Reshaun-AnimeLevelsMaster-2026-1.6.0.zip](https://github.com/user-attachments/files/30925451/Reshaun-AnimeLevelsMaster-2026-1.6.0.zip)

[Reshaun-AnimeTexturePack.zip](https://github.com/reshaun/Neverball-Anime-Gaiden-Levels/files/13324578/Reshaun-AnimeTexturePack.zip)

(For Cutsman)

**Comment by parasti** [comment]
That looks beautiful, Reshaun!

**Reply by reshaun** [reply]
Thanks.
